src/jdk.httpserver/share/classes/com/sun/net/httpserver/HttpPrincipal.java

```diff
@@ -63,41 +68,47 @@ public boolean equals (Object another) {
     }
 
     /**
-     * returns the contents of this principal in the form
-     * <i>realm:username</i>
+     * Returns the contents of this principal in the form
+     * <i>realm:username</i>.
+     *
+     * @return the contents of this principal in the form realm:username
      */
     public String getName() {
         return username;
     }
```

Comment:
Can you log a follow-up bug about this? It seems that the implementation just returns `username`, not `realm:username`.

Reply:
I've created an issue for this as requested: https://bugs.openjdk.java.net/browse/JDK-8255584